presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkLocalShuffleReadInfo.java

```diff
@@ -47,17 +55,31 @@ public int getNumPartitions()
         return numPartitions;
     }
 
+    @JsonProperty
+    public String getQueryId()
```

Comment:
It would be nice to document all these properties.

Reply:
Added the comment on the properties definition line.